hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2315,15 +2315,15 @@ boolean snapshotCleanupSwitch(final boolean on, final boolean synchronous)
    * @param servers set of servers to remove
    * @throws IOException if a remote or network exception occurs
    */
-  void removeRSGroup(Set<Address> servers) throws IOException;
+  void removeServers(Set<Address> servers) throws IOException;
```

Comment:
removeServersFromRSGroup? And no need rsgroup as the parameter?

Reply:
A server can only belong to one group, the the group here is redundant? In the implementation, we will check whether these servers are from the same group.